multi-int-store/Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM openjdk:11-jre-slim
+FROM openjdk:11
```

Comment:
The image openjdk:11 provides better debugging tools than jre-slim. Before deployment to prod, consider changing back to a slim image.